TimoCloud-Universal/src/main/java/cloud/timo/TimoCloud/base/managers/BaseInstanceManager.java

```diff
@@ -234,6 +234,19 @@ private void startServer(BaseServerObject server) {
             File serverProperties = new File(temporaryDirectory, "server.properties");
             setProperty(serverProperties, "online-mode", "false");
             setProperty(serverProperties, "server-name", server.getName());
+            File configFile = new File(temporaryDirectory, "spigot.yml");
+            configFile.createNewFile();
+            Yaml yaml = new Yaml();
+            Map<String, Object> config = (Map<String, Object>) yaml.load(new FileReader(configFile));
+            if (config == null) config = new LinkedHashMap<>();
+            Map<String, Object> settings = (Map<String, Object>) config.get("settings");
+            if (settings == null) settings = new LinkedHashMap<>();
+            settings.put("bungeecord", true);
```

Comment:
We can't force users without any warning to use the bungee mode since this changes the UUID format and can break e.g. permission systems without the users noticing why. 

I think it's best to *read* the value from the spigot.yml and print a warning message if it's false. The user can then change it himself. If we change it ourselves, there must be an option to prevent that.